.github/workflows/benchmark_long.yml

```diff
@@ -0,0 +1,499 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Executes long-running benchmarks that are expected to take multple hours.
+
+name: Benchmark Long
+
+on:
+  schedule:
+    - cron: '0 09 * * *'
```

Comment:
Can you add a comment about what this corresponds to? Every day at 09:00 UTC?

Reply:
Added comment and updated to run twice a day.